resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml

```diff
@@ -1,69 +1,40 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: aks-kubelet-monitoring-kyma-patch
+  name: aks-kubelet-monitoring-cleanup-patch
```

Comment:
You cherry-picked https://github.com/kyma-project/kyma/pull/8978 for this file but you have not processed the second file of the PR. Please double-check that. MIght be OK to cherry-pick it, but please do it proper or skip it.

Reply:
we miss removal of job aks-delete-kubelet-monitoring-kyma-patch but this will take no effect since cleanup job will do same.

Reply:
So you will half-way cherry-pick it? ok, but please make it transparent in the description of the PR.

And the missed aks-delete-patch job has a "helm.sh/hook-delete-policy: hook-succeeded" which will not be fixed as agreed..

Reply:
please let me know if we need to cherry pick aks kubelet fix too. I cherry picked only helm hook fix.